src/domain/app/i18n/fi.json

```diff
@@ -1883,6 +1882,39 @@
       "placeholderConfirmationMessage": "Syötä vahvistusviesti ilmoittautuneille",
       "placeholderEvent": "Valitse tai hae tapahtuma",
       "placeholderInstructions": "Syötä ilmoittautumisohjeet",
+      "registrationAccount": {
+        "buttonReturnDefaults": "Palauta tilin oletusarvot",
+        "helperDefaultValue": "Oletusarvo: {{value}}",
+        "labelAccount": "Kirjanpitotili",
+        "labelBalanceProfitCenter": "Tasapainotettu tulosyksikkö",
+        "labelCompanyCode": "SAP-yrityskoodi",
+        "labelInternalOrder": "Sisäinen tilaus",
+        "labelMainLedgerAccount": "Pääkirjanpitotili",
+        "labelOperationArea": "SAP-toimintoalue",
+        "labelProfitCenter": "Tulosyksikkö",
+        "labelProject": "Projekti",
+        "placeholderAccount": "Valitse kirjanpitotili",
+        "placeholderBalanceProfitCenter": "Syötä tasapainotettu tulosyksikkö",
```

Comment:
```suggestion
        "placeholderBalanceProfitCenter": "Syötä taseen tulosyksikkö",
```

Reply:
I changed this to "Syötä tasetilin tulosyksikkö" to be consistent with the field label text.